### PR TITLE
React to rename of Microsoft.EntityFrameworkCore.Commands and dotnet-ef

### PR DESCRIPTION
**Pull request by mikeharder**
@natemcmaster

### CODE REVIEW

File: src/Benchmarks/project.json [hunk 1]
```diff
@@ -6,7 +6,6 @@
   },
   "dependencies": {
     "Dapper": "1.50.0-*",
-    "Microsoft.EntityFrameworkCore.Commands": "1.0.0-*",
```
Comment (natemcmaster) [inline src/Benchmarks/project.json]: You also need " "Microsoft.EntityFrameworkCore.Tools": "1.0.0-*" in dependencies.

https://docs.efproject.net/en/latest/cli/dotnet.html#installation

Comment (NTaylorMullen) [inline src/Benchmarks/project.json]: And as a build-time only dependency.

**Comment by mikeharder** [comment]
@natemcmaster, @NTaylorMullen: Updated

**Comment by natemcmaster** [comment]
:shipit:

**Pull request merged**